sdk/core/Azure.Core/src/Shared/MutableJsonDocument.ChangeTracker.cs

```diff
@@ -135,6 +137,51 @@ internal IEnumerable<MutableJsonChange> GetRemovedProperties(string path, int hi
                 }
             }
 
+            internal IEnumerable<string> GetChangedProperties()
+            {
+                HashSet<string> unique = new();
+                if (_changes == null)
+                {
+                    return unique;
+                }
+
+                // Get unique properties
+                for (int i = _changes!.Count - 1; i >= 0; i--)
+                {
+                    MutableJsonChange c = _changes[i];
+                    unique.Add(c.Path);
+                }
+
+                // Sort them
+                List<string> list = new(unique);
+                list.Sort();
```

Comment:
This might not work, ads I am not sure List.Sort is a stable sort.

Reply:
Ah, good to know - thank you!

Reply:
I implemented my own routine to retrieve changes in sorted order.  It avoids the HashSet and List allocations and I think should be stable.